bbn-test-5/babylon-validators/README.md

```diff
@@ -0,0 +1,290 @@
+seeds = "8fa2d1ab10dfd99a51703ba760f0ef555ae88f36@16.162.207.201:26656"
+
+# Comma separated list of nodes to keep persistent connections to
+persistent_peers = "8fa2d1ab10dfd99a51703ba760f0ef555ae88f36@16.162.207.201:26656"
+```
+
+In the `config.toml` add in the `seeds` and `persistent-peers` token (this can be the same token).
+
+We next want to add in the `genesis.json` file. To do this, either copy the genesis file from `https://rpc.devnet.babylonlabs.io/genesis` or we can use the terminal with the following command.
+
+```
+wget https://github.com/babylonlabs-io/networks/raw/main/bbn-test-5/genesis.tar.bz2
+tar -xjf genesis.tar.bz2 && rm genesis.tar.bz2
+mv genesis.json ~/.babylond/config/genesis.json
+```
+
+This file needs to overwrite the existing genesis file in the `~/.babylond/config/genesis.json` and please ensure that the `chain-id` is correct. The chain id is what you used in the `babylond init` command above.
+## Setup the required keys for operating a validator 
+
+### Keys for a CometBFT validator
+
+Setting up the key is crucial as it serves as the validator's identity. The key-pair will be used for signing blocks, participating in consensus and managing validator operations. To add a key run the following command:
+
+```shell
+babylond --keyring-backend test keys add <name> --home=./nodeDir
+```
+  
+We use `--keyring-backend test`, which specifies which backend to use for the keyring, `test` stores keys unencrypted on disk. The `<name>` specifies a unique identifier for the key.
+
+The execution result displays the address of the newly generated key and its public key. Following is a sample output for the command:
+
+```shell
+- address: bbn1kvajzzn6gtfn2x6ujfvd6q54etzxnqg7pkylk9
+
+name: test-val
+
+pubkey: '{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"Ayau+8f945c1iQp9tfTVaCT5lzhD8n4MRkZNqpoL6Tpo"}'
+
+type: local
+```
+### Keys for a BLS Voting
+#### What is BLS Voting
+
+BLS (Boneh-Lynn-Shacham) voting is used to create compact checkpoints at the end of each epoch. An epoch is a fixed period within the blockchain, defined or set by a number of blocks, during which a validator set remains consistent. This stability allows Babylon to create and submit checkpoints only once per epoch, reducing the overhead of interacting with the Bitcoin blockchain.
+
+To read more on BLS Voting please see [here](#)
+#### Create BLS Key
+
+Validators are expected to submit a BLS signature at the end of each epoch. To do that, a validator needs to have a BLS key pair to sign information with.
+
+Using your address from the keyring generation run:
+
+```shell
+babylond create-bls-key <address>
+```
+
+Your address should be the one that was generated in the keyring generation step and looks something like this: `bbn1kvajzzn6gtfn2x6ujfvd6q54etzxnqg7pkylk9`.
+
+This command will create a BLS key and add it to the `~/.babylond/config/priv_validator_key.json` that was generated when `init` was run.
```

Comment:
During the init, you used `.nodeDir` as the bbn folder.
We should either re-use it here or convert to `.babylond` everywhere.

Reply:
Fine with the `nodeDir`.